client/src/components/Thread.js

```diff
@@ -70,7 +72,8 @@ class Thread extends React.Component {
 
                     <UpdateThread threadId={this.state.threadId} />
 
-                    <button id={this.state.threadId} onClick={this.props.handleDeleteThreads}>Delete</button>
+                    <button id={this.state.threadId} onClick={event => {this.props.handleDeleteThreads(event);
```

Comment:
does the handledeletethreads function require an event parameter? did you change the handledeletethreads function?

Reply:
nevermind I understand now!